websites/W/Waze/presence.ts

```diff
@@ -0,0 +1,62 @@
+const presence = new Presence({
+    clientId: "854718080433520661"
+  }),
+  presenceData: PresenceData = {
+    largeImageKey: "logo",
+    startTimestamp: Date.now()
+  },
+  pages: {[key: string]: string} = {
+    "/": "Homepage",
+    "/waze-app/": "Mobile App",
+    "/waze/": "Features",
+    "/carpool/": "Waze Carpool",
+    "/events/": "Events",
+    "/partners/": "Partners",
+    "/wazeforcities/": "Waze for Cities",
+    "/product-partners/": "Product Partners",
+    "/ads/": "Waze Ads",
+    "/carpool/companies/": "Carpool Partners",
+    "/editor/": "Map Editor",
+    "/about/": "About Us",
+    "/careers/": "Waze Careers",
+    "/contact/": "Contact Us",
+    "/communities": "Waze Communities",
+    "/legal/tos/": "Terms of Use",
+    "/legal/privacy/": "Privacy Policy",
+    "/legal/copyright/": "Copyright Policy"
+  };
+
+let path: string,
+    start: string,
+    end: string;
+
+function fnd() {
+  path = window.location.pathname;
+  if (!path.endsWith("/")) path = `${path}/`;
+  path = path.replace(/(\/[a-z]{2}|\/[a-z]{2}-[A-Z]{2})\//,"/");
+
+  if (path.includes("/live-map")) {
+    start = end = "";
```

Comment:
eee what? you don't need this line?
```suggestion
```

Reply:
I needed it before, but I changed the code. Maybe I don't need it now.

Reply:
I don't think you need that since you have `let start: string, end: string` already 😂

Reply:
No, it was for clearing the origin and destination.

Reply:
Okay then

Reply:
Tested it now and it is not needed anymore because of how I changed the code.